src/main/java/org/emela/chautari/model/AuthenticationResponse.java

```diff
@@ -1,105 +1,23 @@
 package org.emela.chautari.model;
 
-import java.util.Objects;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import com.fasterxml.jackson.annotation.JsonCreator;
-import io.swagger.annotations.ApiModel;
-import io.swagger.annotations.ApiModelProperty;
-import javax.validation.Valid;
-import javax.validation.constraints.*;
-
-/**
- * AuthenticationResponse
- */
-@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2020-05-10T13:13:33.782-04:00[America/New_York]")
-
-public class AuthenticationResponse   {
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@Builder
+public class AuthenticationResponse {
```

Comment:
generated class?

Reply:
I dont have plugin to generate code from open api, so I will merge this code and you can replace with generated ones?

Reply:
Yes. You can merge now.